site/docs/hosting/vercel.md

```diff
@@ -80,9 +80,24 @@ if (!token) throw new Error("BOT_TOKEN is unset");
 
 const bot = new Bot(token);
 
-export default webhookCallback(bot, "http");
+export default webhookCallback(bot, "std/http");
 ```
 
+::: tip grammY has a limited support for [Vercel Edge Functions](https://vercel.com/docs/functions/edge-functions)
+This means that you can use core grammY package and some plugins, but other may be incompatible due node-specific dependencies that are currently not supported by [Edge Runtime](https://edge-runtime.vercel.app).
+
+Currently, we don't have a comprehensive list of compatible plugins, so you need to test it by yourself.
```

Comment:
Do we need to specify that?

Reply:
Depends on...
- [ ] https://github.com/grammyjs/grammY/issues/372